app-guides/slp-pools.md

```diff
@@ -1,15 +1,23 @@
 ---
-description: How to get yield as a SLP using Angle app
+description: How to earn yield as a SLP using Angle app
 ---
 
-# SLP Pools
+# Deposit Tokens to Earn Yield
 
-Each collateral/stablecoin pair on Angle has a dedicated pool, which funds are invested into yield-earning strategies to put the protocol capital to work and earn interests. You can deposit your collateral on the pools of your choice. Even though some pools have the same tokens, they don't necessarily have the same APY. The reason is explained in the [App FAQ](app-faq.md).
+Each collateral/stablecoin pair on Angle has a dedicated pool, which funds are invested into yield-earning strategies to put the protocol capital to work and earn interests. Users, called Standard Liquidity Providers or SLP, can deposit their tokens into those pools to earn a part of the yield generated by the protocol. You can read more about this in the [SLP page](/concepts/standard-liquidity-providers/README.md). 
+
+You can deposit tokens in the pools of your choice from [this page](https://app.angle.money/#/slp):
 
 1. Choose the pool you would like to deposit your tokens in and click on `Deposit`.
-2. On the deposit page, enter the amount of tokens you want to put in the pool and confirm the transaction. You might need to approve the pool if it hasn't been done before.
-3. You will receive sanTokens in exchange, which represent your liquidity in the pool and automatically accrue interest.
-4. To withdraw your funds, just click on the `Withdraw` button on the Pools page, and input the number of sanTokens you want to send back.
+2. On the deposit page, enter the amount of tokens you want to put in the pool. You might need to approve the token through a transaction or a signature. 
+3. You now have two options: `Deposit`, and `Deposit and Stake`. 
+    - **`Deposit`** will send your collateral to the pool, and you will receive the associated sanTokens, which automatically accrue interest. 
+    - **`Deposit and Stake`** will deposit your collateral into the protocol, generate the associated sanTokens, and stake them automatically in the related gauge on the [DAO App](https://dao.angle.money/#/stake). You will start accumulating ANGLE rewards on top of the interest of the sanTokens. 
```

Comment:
You are not staking on the DAO App: you are staking in the liquidity gauge which you can track on the DAO App